src/config.ts

```diff
@@ -0,0 +1,38 @@
+import { Container } from 'aurelia-dependency-injection';
+import { Validator } from './validator';
+import { StandardValidator } from './implementation/standard-validator';
+import { validateTrigger } from './validate-trigger';
+
+/**
+ * Aurelia Validation Configuration API
+ */
+export class GlobalValidationConfiguration {
+  public static DEFAULT_VALIDATION_TRIGGER = validateTrigger.blur;
+
+  private validatorType: { new (...args: any[]): Validator } = StandardValidator;
+  private validationTrigger = GlobalValidationConfiguration.DEFAULT_VALIDATION_TRIGGER;
+
+  /**
+   * Use a custom Validator implementation.
+   */
+  public customValidator(type: { new (...args: any[]): Validator }) {
+    this.validatorType = type;
+  }
+
+  public defaultValidationTrigger(trigger: validateTrigger) {

```

Comment:
Does getter/setter make more or less sense than this combo?

Reply:
My idea here was to do the configuration in chainable methods, like aurelia-framework config does. I see I forgot to add `return this` to the setter though.

I don't have a strong preference. If you prefer, I can change this to a setter/getter.

Reply:
Note though that there was already an existing method `public customValidator(type)` so this setter is consistent with that.

Reply:
I see, I think fluent will be nicer.

Reply:
I'll have the setters `return this` then.